src/wallet/db.cpp

```diff
@@ -419,18 +412,8 @@ bool BerkeleyBatch::VerifyDatabaseFile(const fs::path& file_path, std::vector<bi
     if (fs::exists(walletDir / walletFile))
     {
         std::string backup_filename;
-        BerkeleyEnvironment::VerifyResult r = env->Verify(walletFile, recoverFunc, backup_filename);
-        if (r == BerkeleyEnvironment::VerifyResult::RECOVER_OK)
-        {
-            warnings.push_back(strprintf(_("Warning: Wallet file corrupt, data salvaged!"
-                                     " Original %s saved as %s in %s; if"
-                                     " your balance or transactions are incorrect you should"
-                                     " restore from a backup."),
-                walletFile, backup_filename, walletDir));
-        }
-        if (r == BerkeleyEnvironment::VerifyResult::RECOVER_FAIL)
-        {
-            errorStr = strprintf(_("%s corrupt, salvage failed"), walletFile);
+        if (!env->Verify(walletFile)) {
```

Comment:
nit: How about returning the `verify` result so we can log the error:
https://docs.oracle.com/cd/E17276_01/html/api_reference/C/dbverify.html

Reply:
I don't think the results of `verify` are meaningful.